apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -222,6 +250,28 @@ int recovery_file(char *src, char* backup, char* backup_dir, int bufsize, int id
 			return ERROR;
 		}
 
+		/* Extend last file same as backup file, it's not common usage but add it to test */
```

Comment:
Maybe better to test extendfile() separately only...

Reply:
Mixing is more complicated case, I modified it to test append too.